src/latch_cli/services/launch_v2/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Service to launch a workflow."""
```

Comment:
can you just move the main launch fn into this module and call it `launch` instead of `launch_v2`